### DISCUSSION TITLE
Toggle helpful feedback

### DISCUSSION THREAD

**Discussion by nashysolutions** [pointfreeco/vapor-routing#30] [Q&A]
It would be ideal if I could disable this helpful feedback once I deploy to production, for security purposes. Is that possible?

```
[ INFO ] error: unexpected input
 --> input:1:2
1 | /1/users
  |  ^ expected "v1/users"
```

**Comment by stephencelis** [comment]
@nashysolutions That error should only show up when your server is running in development:

https://github.com/pointfreeco/vapor-routing/blob/ae1db2ec96fad88b00173a265313de2c447a9945/Sources/VaporRouting/VaporRouting.swift#L52-L53

Are you seeing otherwise?

**Reply by nashysolutions** [reply]
No I haven't. Good to know. Thanks.
